source/2020-05-01-the-ember-times-issue-146.md

```diff
@@ -59,6 +60,21 @@ Want to get started? Be sure to read the [official status update for the Ember W
 
 ---
 
+## [ember-bootstrap 4 Release Candidate 🛳](https://twitter.com/simonihmig/status/1255099221415510016)
+
+The folks at [kaliber5 @kaliber5](https://github.com/kaliber5) have put out a new 4.0.0-rc.0 release candidate for [ember-bootstrap](https://github.com/kaliber5/ember-bootstrap)! 🎉
```

Comment:
Optional: I think we can omit the specific version number to make the sentence sound more natural.
```suggestion
The folks at [kaliber5](https://github.com/kaliber5) have put out a new release candidate for [ember-bootstrap](https://github.com/kaliber5/ember-bootstrap)! 🎉
```